lessons/Early-Math/translating/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+
+- Every Circle must have one - and only one! - function, written at the top.
+
+- The inputs of the function are written left to right, in the middle of the Circle.
+
+Share these two key concepts before explaining the following instructions.
+
+[.lesson-instruction]
+- With your partner, match each Expression card with the corresponding Circle card. Do not sort the Words cards yet.
+- There are more Circles than expressions. Create a separate pile for any cards that do not have a match.
+- Lay the cards out on the table in front of you so you can clearly see both the Circle and the expression. Create a separate pile for any cards that do not have a match.
+- Discuss any questions that arose.
+
+Circulate as students sort their cards, ensuring that they are carefully analyzing each card so that they see the connection between the arithmetic expression and the Circle of evaluation. This first phase of matching cards should be relatively quick - but it might raise questions on a few topics:
+
+- *Symbols*. Circles utilize * to represent multiplication and / to represent division. (Why? Circles are a bridge representation - one which can eventually be used to help students learn to code! Although we use these symbols on all of our materials, you and your students can use whichever operator symbols are most comfortable.)
+
+- *Order of terms*. While @math{5 + 3} and @math{3 + 5} both evaluate to the same _answer_, they are not the same _expression_.
+
+- *Position of the operator*. The operator always belongs at the top of the Circle, and not in between terms.
+
+[.lesson-instruction]
+- Now, you’re going to receive a set of cards with expressions written out in _words_.
+- One at a time, take turns reading each Words card out loud. After reading the card, place it with the appropriate Circle and Expression. Some sets will include just 3 cards, and others will have more.
+- Explain to your partner how and why you placed each card.
+- You and your partner must agree on each card’s placement before advancing to the next.
+
+Again, circulate during the activity. As students match expressions in words to their arithmetic expressions and Circles, additional challenges may arise. In particular:
+
+- *One-third of 15*. This card will likely motivate some discussion. Take advantage of opportunities to help students connect multiplication and division as inverse operations. Explain to students that another way to represent this expression is @math{\frac{1}{3} \times 15}. Dividing by 3 produces the same outcome as multiplying by @math{\frac{1}{3}}.
+- *Translating subtraction.* Students often translate "3 less than 15" into @math{3 - 15}, rather than @math{15 - 3}. Discuss the meaning of the word "less" in this context. Sometimes, asking, “What value is 3 less than 15?” can help students to make the connection.
```

Comment:
If students have already been introduced to inequality statements (3 < 5), there is often another point of confusion worth referencing here:
- "is less than" is a comparison statement (3 < 5)
- "less than" without the "is" indicates subtraction 
- "than" is a word that tends to force the numbers to appear in the opposite order in which the language writes about them
- "is" is a statement of inequality / translates to equals

Reply:
this is great! Thanks.